Autopilot/Path Manager/nbproject/configurations.xml

```diff
@@ -123,6 +127,9 @@
         <property key="use-cci" value="false"/>
         <property key="use-iar" value="false"/>
       </C30>
+      <C30-AR>
+        <property key="additional-options-chop-files" value="false"/>
+      </C30-AR>
```

Comment:
On chris's not, this configuration file is fine, since it adds files to the project